docs/content/latest/contribute/core-database/build-from-src-ubuntu.md

```diff
@@ -46,56 +46,32 @@ CentOS 7 is the recommended Linux development and production platform for Yugaby
 
 {{< /note >}}
 
-## Installing necessary packages
+## Install necessary packages
 
 Update packages on your system, install development tools and additional packages:
 
 ```sh
 sudo apt-get update
-sudo apt-get install uuid-dev libbz2-dev libreadline-dev maven ninja-build
+sudo apt-get install uuid-dev libbz2-dev libreadline-dev maven ninja-build \
+                     cmake curl rsync python-pip zip autoconf libtool \
```

Comment:
Ubuntu 16.04 installing cmake this way got version 3.3, but the build checks that it is at least 3.7, so I had to install cmake separately.